clientv3/balancer.go

```diff
@@ -30,7 +30,8 @@ import (
 var ErrNoAddrAvilable = grpc.Errorf(codes.Unavailable, "there is no address available")
 
 // simpleBalancer does the bare minimum to expose multiple eps
-// to the grpc reconnection code path
+// to the grpc reconnection code path.
+// It only keeps one connection open at a time.
```

Comment:
probably don't need to call out it only opens one connection at a time since there's no other balancer yet; it's an implementation detail that may be subject to change (e.g., supporting `n` open connections at a time)